server/controllers/inventory/inventory/core.js

```diff
@@ -250,15 +255,21 @@ async function getItemsMetadata(params) {
       ig.sales_account, ig.stock_account, ig.donation_account, inventory.sellable, inventory.note,
       inventory.unit_weight, inventory.unit_volume, ig.sales_account, ig.stock_account, ig.donation_account,
       ig.cogs_account, inventory.default_quantity, ig.tracking_consumption, ig.tracking_expiration,
-      inventory.importance, t.name AS tag_name, t.color AS tag_color,
+      inventory.importance,
+      (SELECT GROUP_CONCAT(BUID(t2.uuid), ';', t2.name, ';', t2.color
```

Comment:
Do you really need a nested select?  What happens if you just call `GROUP_CONCAT()` in the regular select statement?

Reply:
Good catch.  I was able to get this working without the nested select.